Activities/Credentials/UiPath.Credentials.Activities/AddCredential.cs

```diff
@@ -1,28 +1,43 @@
 ﻿using CredentialManagement;
 using System.Activities;
 using System.ComponentModel;
+using System;
+using System.Activities.Validation;
+using System.Diagnostics;
+using System.IO;
+using UiPath.Credentials.Activities.Properties;
 
 namespace UiPath.Credentials.Activities
 {
-    [Category("System.Credentials")]
```

Comment:
this is the place in tree where the activities are placed by studio after load. Not sure it's safe to remove it. We possibly need a LocalizedCategory attribute

Reply:
As dicussed, changes added to DesignerMetadata in latest commit.